core/server/data/schema/default-settings.json

```diff
@@ -353,6 +353,14 @@
         "mailgun_base_url": {
             "defaultValue": null,
             "type": "string"
+        },
+        "email_track_opens": {
```

Comment:
Prefixing with `email` rather than `mailgun` because it's more related to general email configuration rather than Mailgun specific. The Mailgun bulk-email provider can use the setting to toggle the Mailgun-specific headers when sending email.